pemFioi/quickpi/blocklyQuickPi_lib.js

```diff
@@ -8826,10 +8832,65 @@ var getContext = function (display, infos, curLevel) {
         context.runner.waitDelay(callback, millis);
     };
 
-    context.quickpi.getTemperature = function(location, callback) {
-        var retVal =  25;
 
-        context.waitDelay(callback, retVal);
+    // TODO: change url to the "prod" one
+    context.quickpi.getTemperatureFromCloudUrl = "https://mapadev.com/nicolastest/weather.php";
```

Comment:
I don't think this variable should be put publicly in the context. Just `var getTemperatureFromCloudUrl`.

Same for some other variables.

Reply:
All the variable used by the function has been passed locally